### PR TITLE
docs: add Truly-UI to resources

### PR DESCRIPTION
**Pull request by abalad**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [  X ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ x ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the new behavior?
Add the component library to the list of resources

## Does this PR introduce a breaking change?
```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -395,7 +395,13 @@
...
             "title": "Essential JS 2",
             "url": "https://www.syncfusion.com/products/angular-js2"
-          }
+          },
+          "trulyui": {
+            "desc": "TrulyUI is an Angular UI Framework especially developed for Desktop Applications based on Web Components using the greatest technologies of the world.",
+            "rev": true,
+            "title": "Truly UI",
+            "url": "http://truly-ui.tk"
+           }
```
Comment (gkalpak) [inline aio/content/marketing/resources.json]: Incorrect indentation.

**Review by gkalpak — COMMENTED** [review]
Can you please change the commit message to follow our [commit message guidelines][1]. E.g.:

> docs: add Truly-UI to resources

[1]: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

**Review by StephenFluin — APPROVED** [review]
Content looks good to me once the indentation and commit message are fixed.

File: aio/content/marketing/resources.json (hunk 1 above)
Comment (abalad) [reply aio/content/marketing/resources.json]: I'll do the correction right now.

**Review by StephenFluin — CHANGES_REQUESTED** [review]
The content and indentation looks good now, but you'll need to squash your two commits into one with the right commit message.

**Comment by StephenFluin** [comment]
@abalad I made a tweet showing how you could have fixed this PR for future reference: https://twitter.com/stephenfluin/status/1009904095073718275

**Pull request closed without merging**